_data/governments.yml

```diff
@@ -303,8 +303,10 @@ Hungary:
   - fphgov
 
 India:
+  - DataPortalIndia
   - ISRO
   - itschool
+  
```

Comment:
Could you delete this extra line? Thanks!

Reply:
Deleted, thanks!